app.py

```diff
@@ -63,7 +63,7 @@ def weather(city, state):
     #print out a statement with the current weather info + location that was used/detected
     #return("Right now in "+ city + ", " + state + " it is " + temperature +  degree_sign + " and there  is a " + RAIN_WARNING)
     location = {'city': city, 'state': state}
-    weather_info = {'temperature' : temperature, 'rain' : rain_commentary}
+    weather_info = {'temperature' : temperature, 'rain' : rain_commentary, 'celsius'=config.celsius}
```

Comment:
Also, when Celsius doesn't exist in Config (which is a setup that should be supported) the following error will be thrown:
```AttributeError: module 'config' has no attribute 'celsius'```

Are you able to write this in a way that can handle missing celsius config value?

Reply:
I can fix this too. No problem